src/sentry/replays/query.py

```diff
@@ -531,6 +531,9 @@ def _sorted_aggregated_urls(agg_urls_column, alias):
         "activity",
     },
```

Comment:
Can we insert the `allow_free_text_email=True` here instead of creating a new config?